src/platform-integration/web/wasm.md

```diff
@@ -10,19 +10,27 @@ The Flutter and Dart teams are excited to add
 [WebAssembly](https://webassembly.org/) as a compilation target when building
 applications for the web.
 
-{{site.alert.warning}}
-  Development of WebAssembly support for Dart and Flutter remains ongoing.
-  
-  You can preview Wasm and WebAssembly garbage collection (WasmGC)
-  on the [`master` channel][].
-  Flutter is previewing these features.
-  Ongoing development might result in frequent changes.
+{{site.alert.note}}
+  Development of WebAssembly support for Dart and Flutter remains ongoing,
+  which might result in frequent changes. 
```

Comment:
nit: "might result in frequent changes" feels odd to me. How about "will potentially result in frequent changes"?

Reply:
Done!